mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -470,4 +470,36 @@ class StatefulHomeTest {
       rule.onNodeWithContentDescription(strings.arContentDescription).assertExists()
     }
   }
+
+  @Test
+  fun given_visitedProfileScreen_when_cancelButtonClicked_then_socialScreenIsDisplayed() = runTest {
+    val auth = buildAuth { user("email@example.org", "password", "1") }
+    val store = buildStore {
+      collection("users") {
+        document("1", ProfileDocument("1", name = "A"))
+        document("2", ProfileDocument("2", name = "B"))
+      }
+      collection("games") {
+        document(
+            "id", ChessDocument(uid = "45", whiteId = "1", blackId = "2", moves = listOf("e2-e4")))
+      }
```

Comment:
I think it would be better to not have an unnecessary game, to have as little as possible outside of the tested feature to be tested. (Watch out for formatting with the suggestion)
```suggestion
```

Reply:
Yes